### PR TITLE
Create help

### PR DESCRIPTION
**Pull request by wadtech**
Had a go at #126.

It's just a simple regex looking for --help in the create call. 

It's got a passing spec, let me know if I'm way off the mark.

Cheers

### CODE REVIEW

File: lib/tugboat/cli.rb [hunk 1]
```diff
@@ -141,6 +141,11 @@ def ssh(name=nil)
                    :desc => "Enable backups on the droplet"
 
     def create(name)
+      if name =~ /^(--)?help/
```
Comment (petems) [inline lib/tugboat/cli.rb]: Can we change this to `^--help$|-h$`?

File: spec/cli/create_cli_spec.rb [hunk 2]
```diff
@@ -29,8 +29,41 @@
...
+  -k, [--keys=KEYS]           # A comma separated list of SSH key ids to add to the droplet
+  -p, [--private-networking]  # Enable private networking on the droplet
+  -b, [--backups-enabled]     # Enable backups on the droplet
+  -q, [--quiet]               
+
+Create a droplet.
+eos
+
+      @cli.create('help')
+      expect($stdout.string).to eq help_text
```
Comment (petems) [inline spec/cli/create_cli_spec.rb]: What if someone does want to make a droplet called help though? Unlikely, but still...

Comment (wadtech) [inline spec/cli/create_cli_spec.rb]: I guess the core issue is that it's easy to call create incorrectly and that has billing implications. You could default create to a dry-run mode then pass a parameter to actually go through with the change (though that breaks the current interface)

Comment (petems) [inline spec/cli/create_cli_spec.rb]: Yeah, this is a QOL life change that I like. I'll open a separate PR later to add the edge case for a node called `help`

**Pull request merged**